ivy/functional/frontends/torch/tensor.py

```diff
@@ -1800,6 +1800,24 @@ def isnan(self):
     def lcm(self, other, *, out=None):
         return torch_frontend.lcm(self, other, out=out)
```

Comment:
torch frontend tensor already has lcm implemented, isn't this redundant?

Reply:
actually this is the inplace version of the lcm function, you can check that in the torch docs.